functions/EmuScripts/emuDeckBigPEmu.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+#variables
+BigPEmu_emuName="BigPEmu (proton)"
+BigPEmu_emuType="windows"
+BigPEmu_emuPath="${romsPath}/Applications/BigPEmu/BigPEmu.exe"
+BigPEmu_appID=
+BigPEmu_appData="${HOME}/.steam/steam/steamapps/compatdata/${BigPEmu_appID}/pfx/drive_c/users/steamuser/AppData/Roaming/BigPEmu"
+BigPEmu_BigPEmuSettings="${BigPEmu_appData}/BigPEmuConfig.bigpcfg"
+
+#cleanupOlderThings
+BigPEmu_cleanup(){
+	echo "NYI"
+}
+
+#Install
+BigPEmu_install(){
+	setMSG "Installing $BigPEmu_emuName"
+
+	local showProgress="$1"
+    if wget -m -nd -A "BigPEmu_*.zip" -O "$HOME/Applications/BigPEmu.zip" "https://www.richwhitehouse.com/jaguar/index.php?content=download"; then
```

Comment:
sorry to add another snag - but i notice you used wget here instead of curl.

As insane as this sounds - i have encountered decks without wget. it bit me when we first rolled out the yuzu appimage.

Reply:
So, I tried finding a way to use cURL to do this, but because the download URL is inconsistently named with each release, I needed to use wget's ability to use wildcards to make this function work dynamically.

Everything I found online says to use wget instead of curl for this anyway, so I'm curious if the decks without wget can be easily fixed by telling users to update?

Or if you know a way to dynamically find download URLs with curl, I'm all ears.

Reply:
thoughts on this? I modified trhe code from scraping cemu.info. seems happy.

```
releasesStr=$(curl -sL https://www.richwhitehouse.com/jaguar/index.php?content=download | awk 'BEGIN{
            RS="</a>"
            IGNORECASE=1
            }
            {
            for(o=1;o<=NF;o++){
                if ( $o ~ /href/){
                gsub(/.*href=\042/,"",$o)
                gsub(/\042.*/,"",$o)
                print $(o)
                }
            }
            }' | grep builds)
```

```
echo $releasesStr
https://www.richwhitehouse.com/jaguar/builds/BigPEmu_v106.zip
```

Reply:
Quick testing shows this works in both bash and sh but not zsh (weird). This should probably work.